featurebyte/session/base_spark.py

```diff
@@ -400,6 +400,9 @@ async def register_missing_functions(self, functions: list[dict[str, Any]]) -> N
         # To ensure functionality is updated for a function we should create a new class
         # and re-register the function with the new class
         udf_functions = [
+            ("VECTOR_AGGREGATE_MAX", "com.featurebyte.hive.udf.VectorAggregateMax"),
+            ("VECTOR_AGGREGATE_SUM", "com.featurebyte.hive.udf.VectorAggregateSum"),
+            ("VECTOR_AGGREGATE_AVG", "com.featurebyte.hive.udf.VectorAggregateAverage"),
```

Comment:
Would be good to add some integration tests for these udfs to make sure they work when calling from featurebyte.

Reply:
yeap will be adding in a follow-up PR!